packages/dataviews/src/view-table.js

```diff
@@ -560,9 +564,11 @@ function ViewTable( {
 												field.maxWidth || undefined,
 										} }
 									>
-										{ field.render( {
-											item,
-										} ) }
+										<span className="dataviews-view-table__cell-content-wrapper">
```

Comment:
I now realised that we cannot have an inline element([span](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span#technical_summary)) as a wrapper because it creates invalid html.

The addition of styling this as `display:flex` too, make the previews not render because of the width calculation. 

Can we achieve what this PR does without spans and make sure the previews are displayed properly?

Sorry for not catching this earlier..

Reply:
We actually have an [issue](https://github.com/WordPress/gutenberg/issues/57597) to remove the template previews in Table and List layouts. They're just too small to be effective in those layouts. Does that affect things here at all?

The wrapping _element_ doesn't really matter I think... would a `div` work? Or are you suggesting to remove the wrapper entirely? I couldn't think of a way to achieve the desired result (see the "After" screenshot) without one, did you have an idea?